content/news/XXXX-XX-XX-russians-war.md

```diff
@@ -0,0 +1,18 @@
+title: Russian's war on Ukraine
+authors: Daniel Schürmann
+tags: russian ukraine war
+status: draft
+comments: yes
+
+Since 2022-02-24 we receive horrible news from Ukaine, which have become even worse since then. As an organization committed to freedom we feel the urge speak up and show solidarity for Ukraine.
+
+Russia's brutal invasion of Ukraine cannot be justified by anything and has already been condemned by the UN as a violation of international law. The attack creates endless suffering for men, women and children in Ukraine but also for the many Russian soldiers who are sent into a senseless war that they do not want, do not understand and are not even allowed to call it as such.
```

Comment:
I recommend to omit any kind of biased or judgemental wording like "brutal" or "horrible". Accusations in any direction will only divide people. That's not what we want to achieve.

Reply:
OK, I have removed "brutal"